fix(install): remove escapes on heading when coloring is not supported

paulozulato opened the pull request:
<!-- Thank you for sending a pull request :heart: -->

## Current behavior

Escape characters are still being printed on heading even when terminal doesn't support coloring, resulting in strange characters being printed:
```
\033[0;33m   GET STARTED:\033[0m                                                                                                                                                                                                                            
   Ockam Command is installed at: "$HOME/.ockam/bin/ockam"     
```

## Proposed changes

Removed escape characters in case that colorizing is not supported:
```
   GET STARTED:
   Ockam Command is installed at: "$HOME/.ockam/bin/ockam"
```

## Checks

<!-- To help us review and merge this pull request quickly, please confirm the following:  -->

- [x] All commits in this Pull Request are [signed](https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits) and Verified by Github.
- [x] All commits in this Pull Request follow the Ockam [commit message convention](https://github.com/build-trust/.github/blob/main/CONTRIBUTING.md#commit-messages).
- [x] There are no Merge commits in this Pull Request. The Ockam repo maintains a linear commit history so we merge PRs by rebasing them to the develop branch. Rebasing to the latest develop branch and force pushing to your PR branch is okay.
- [x] I accept the Ockam Community [Code of Conduct](https://github.com/build-trust/.github/blob/main/CODE_OF_CONDUCT.md).
- [x] I have accepted the Ockam [Contributor License Agreement](https://github.com/build-trust/ockam-contributors/blob/main/CLA.md) by adding my Git/Github details in a row at the end of the [CONTRIBUTORS.csv](https://github.com/build-trust/ockam-contributors/blob/main/CONTRIBUTORS.csv) file in a separate pull request to the [build-trust/ockam-contributors](https://github.com/build-trust/ockam-contributors) repository. The easiest way to do this is to [edit the CONTRIBUTORS.csv](https://github.com/build-trust/ockam-contributors/edit/main/CONTRIBUTORS.csv) file in the github web UI and create a PR, this will mark the commit as verified.

<!-- Looking forward to merging your contribution!! -->

metaclips commented on install.sh:
```diff
@@ -85,7 +85,7 @@ heading() {
     printf "${_orange}   $1${_no_color}\n"
   else
-    echo "${_orange}   $1${_no_color}"
+    echo "   $1"
```
printf should handle this correctly
```bash
printf "${_orange} $1${_no_color}"
```

metaclips reviewed (dismissed):
Thanks for an amazing PR

metaclips committed:
style: remove ansi color codes from unsupported environments during install

For environments that don't support ANSI Select Graphic Rendition sequenes,
the heading would still display the raw code like so:
\033[0;33m   GET STARTED:\033[0m
This commit updates the heading function in the install script to prevent that
raw output.

Signed-off-by: outerwear <michaelcoates@protonmail.com>
Co-authored-by: Paulo Zulato <paulozulato@gmail.com>
Co-authored-by: Michael Coates <michaelcoates@protonmail.com>

metaclips commented:
There's a PR that adds same fix https://github.com/build-trust/ockam/pull/5966, I made both PRs co-authors.

mrinalwadhwa commented:
@paulozulato we landed the change as part of https://github.com/build-trust/ockam/pull/5966, congratulations on an awesome first contribution.

paulozulato commented:
thank you very much, @mrinalwadhwa
